### PR TITLE
Unit Test - Test view invite view model

### PR DESCRIPTION
**Pull request by at-toannguyen1**
#### Related issues (Required)
- Task #202 
- Bug #

#### Description (Required)
- [x] Unit Test view invite view model

### CODE REVIEW

File: app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    fun initTest() {
+        userId = ""
+        MockitoAnnotations.initMocks(this)
+        viewModel = ViewInviteViewModel(userId, groupRepository)
+    }
+
+    @Test
+    fun `Given invites of user - When call get invites of user - then return right invite`() {
+        val inviteTest = TestObserver<Invite>()
+        val invite = Invite("", "", "", true)
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt]: change this line to `val invite = Invite("from", "to", "groupName", true)`

File: app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        `when`(groupRepository.getInvite(TestUtil.any())).thenReturn(Observable.just(invite))
+        viewModel.getInvitesOfUser().subscribe(inviteTest)
+        inviteTest.assertValue { it == invite }
+    }
+
+    @Test
+    fun `Given remove invite user - When call remove invite user from group - then return progressbar show then hide`() {
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt]: change this line to `fun Given a invite - When call remove invite user from group - then return progressbar show then hide() {`

File: app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        val statusTest = TestObserver<Boolean>()
+        val invite = Invite("", "", "", true)
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt]: change this line to `        val invite = Invite("from", "to", "groupName", true)
`

File: app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        `when`(groupRepository.deleteUserInvite(TestUtil.any(), TestUtil.any())).thenReturn(Single.just(true))
+        viewModel.progressDialogObservable.subscribe(statusTest)
+        viewModel.removeInviteUserFromGroup(invite).subscribe()
+        statusTest.assertValues(true, false)
+    }
+
+    @Test
+    fun `Given accept invite user - When call accept invite - then return progressbar show then hide`() {
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt]: chang this line to `    fun Given a invite user - When call accept invite - then return progressbar show then hide() {
`

File: app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        val statusTest = TestObserver<Boolean>()
+        val invite = Invite("", "", "", true)
+        `when`(groupRepository.acceptInvite(TestUtil.any(), TestUtil.any())).thenReturn(Single.just(true))
+        viewModel.progressDialogObservable.subscribe(statusTest)
+        viewModel.acceptInvite(invite).subscribe()
+        statusTest.assertValues(true, false)
+    }
+}
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/ViewInviteViewModelTest.kt]: add break line

**Comment by at-vinhhuynh** [comment]
@at-tienhoang 
I finish review this PR, please check it again.
thank you!

**Comment by at-dev-support-01** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">All test passed. (6/6)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**